drizzlepac/devutils/comparison_tools/read_hla/read_hla_image.py

```diff
@@ -0,0 +1,59 @@
+"""read_hla_image.py: read an HLA image (from a file or from HLA webservice) and correct WCS
+
+R. White, 2019 June 20
+"""
+
+from __future__ import print_function
+import omegaxyz, os
+from astropy.io import fits
+
+def read_hla_image(dataset, applyomega=True, verbose=False,
+        url="https://hla.stsci.edu/cgi-bin/getdata.cgi"):
+
+    """Return a astropy.io.fits HDUlist for the given HLA dataset.
+    
```

Comment:
Pep8 docstring - parameter description.

Reply:
Issue resolved in commit [041e699](https://github.com/spacetelescope/drizzlepac/pull/548/commits/041e69974fe95fc578d00d35c63dc4bcb8eb88fa)